WireCalling/Tests/WireCallingTests/WireMeetings/PresentationTests/CreateInstantMeetingViewModelTests.swift

```diff
@@ -0,0 +1,243 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import SwiftUI
+import Testing
+@testable import WireCallingUI
+@testable import WireReusableUIComponents
+
+@Suite("CreateInstantMeetingViewModel Tests")
+struct CreateInstantMeetingViewModelTests {
+
+    private let mockPasswordValidator: MockPasswordValidator
+    private let viewModel: CreateInstantMeetingViewModel
+
+    init() {
+        self.mockPasswordValidator = MockPasswordValidator()
+        self.viewModel = CreateInstantMeetingViewModel(
+            accentColor: .blue,
+            passwordValidator: mockPasswordValidator
+        )
+    }
+
+    // MARK: - isNextButtonEnabled Tests
+
+    @Test("isNextButtonEnabled is false with empty title")
+    func isNextButtonEnabled_EmptyTitle() {
```

Comment:
very optional suggestion: If you make wire calling use swift 6.2 (search "// swift-tools-version: 6.1") you can do:
```suggestion
    @Test
    func `isNextButtonEnabled is false with empty title`() {
```

Reply:
Cool